docs/faq.md

```diff
@@ -88,3 +88,9 @@ influence their behavior. While Black does its best to recognize such comments a
 them in the right place, this detection is not and cannot be perfect. Therefore, you'll
 sometimes have to manually move these comments to the right place after you format your
 codebase with _Black_.
+
+## Can I run black with PyPy?
+
+Yes, there is _experimental_ support for PyPy 3.7 and higher. You cannot format Python 2
+files under PyPy, because PyPy's inbuilt ast module does not support this. For the most
+stable user-experience, we recommend using CPython if possible.
```

Comment:
```suggestion
stable user experience, we recommend using CPython if possible.
```
I'm also not too happy with this sentence. If we don't have concrete reasons to say PyPy is unstable, it feels like FUD.

Reply:
I'll remove this sentence - if you're confident in your test suite then happy to just go with "it works" 🙂